tests/Database/DatabaseEloquentCollectionTest.php

```diff
@@ -202,14 +202,6 @@ public function testCollectionReturnsUniqueItems()
 	}
 
 
-	public function testLists()
```

Comment:
That may be true, but that's not the point of the tests. If the implementation changes in the future, we'd still want this test there, even if it's technically testing the same thing at the moment.